docs/get-started.md

```diff
@@ -28,8 +28,20 @@ docker pull vesoft/nebula-graph:latest
 to get `nebula` image and `docker images` that can display images status.
 
 If `docker` is slow when pulling the image, configure an accelerator.
```

Comment:
"If it's too slow to pull the Docker image, you could configure your own preferred registry mirrors".

I don't know whether the word **accelerator** is a convention in the Docker world or not, but I think you are inventing one. Usually, it's called mirrors.

Besides, Docker, as a product, just like Github and Nebula Graph, we usually don't style them monospace.

Reply:
fixed